### ISSUE TITLE
implementation for the segmentation? 

### ISSUE THREAD

**Issue by apple2373** [durandtibo/wildcat.pytorch#1]
Hi, 

I could not find the implementation for the segmentation and its evaluation.... Where is it? 

Best,
Satoshi

**Comment by ghost** [comment]
@apple2373 
this may be helpful for you
https://github.com/lucasb-eyer/pydensecrf

**Comment by KeyKy** [comment]
I also want to learn the segmentation code~

**Comment by SII-Ferenas** [comment]
Actually you have to implement your seg masks based on the paper by yourself. After trained from datasets, the net can be directluy used for the mask Inference. The output of the class-wise pooling (attention, the code writer links the class-wise and spatial-wise into one pooling module by Sequentail, so you have to split them in the net to get the class-wise pooling, whose outputs are 20*7*7 in VOC12) , then you choose the maxscore in each channel(classes) to generate the final seg masks

**Comment by liuyue718** [comment]
Hello, the segmentation mask I got is 1/32 of the size of the input image. To get the segmentation result of the same size as the input image, do I need to directly sample the original size of the image?Isn't that a lot of resolution lost
